docs/pages/enroll-resources/machine-id/troubleshooting.mdx

```diff
@@ -89,15 +96,15 @@ version: v2
 $ tctl rm lock/5cee949f-5203-4f3b-9805-dac35d798a16
 ```
 
-Next, reset the generation counter by deleting and recreating the bot:
+Next, use `tctl bots instances add` to generate a new join token for the bot:
 ```code
-$ tctl bots rm example
-
-$ tctl bots add example --roles=foo,bar
+$ tctl bots instances add example
```

Comment:
Perhaps could be clearer that "example" here is the name of the bot ? Perhaps a short sentence before the instructions like "The following steps show rectifying this issue for a bot named "example""

Reply:
Hmm, good point, we never called out the name of the bot. I've adjusted the text for the whole entry to make that clear in addition to repeating it here.